### ISSUE TITLE
Need to determine source

### ISSUE THREAD

**Issue by karindalziel** [CDRH/earlywashingtondc#10]
I need guidance on how to format the source field. 

It would be most helpful if this was written out in rules I can then program, i.e. 

Ideal: 

National Archives and Records Administration, Record Group 21.6, Box 705, Folder 132

Rules: 

National Archives and Records Administration - [from sourceDesc/bibl/repo](If none, do not show this or comma after)
Record Group 21.6 [from sourceDesc/bibl/idno](If none, do not show this or comma after)
Box 705, Folder 132 - ????

Basically, I need a list of all the fields I might be pulling from, in which order, and how they might be formatted. Some docs don't have box or folder, and I'm not sure if there is anything else I should be looking for. 

In the example Kaci gave:

https://docs.google.com/document/d/19VxCVX04hVexVqI1eftw1HKsmLJ0fp47q0vrV4a95yk/edit

I am not sure where she pulled "Box 705, Folder 132" in the source field.
